src/baseSelectors.js

```diff
@@ -22,17 +22,17 @@ export const getCurrentRouteParams = createSelector(
 
 export const getTopicListScreenParams = createSelector(
   getCurrentRouteParams,
-  params => params || { streamId: -1 },
```

Comment:
This was fixed by using the `-1` value instead.

Reply:
But I thinks `undefined` is better choice.
Because in future if we change `-1` to something else, then we need to change them at multiple places (two changes in https://github.com/zulip/zulip-mobile/commit/9302bf9dd10fea75a07c2816ac3ed4b32fd73f19).
(or may be in future we start using `-1` as some flag)
But `undefined` will always be `undefined`.

Reply:
ya! got it. 
I was thinking to make it generic. (But in this scenario, it is also used in `topicSelector`. So then we need to add extra check in that. And will eventually become dirty if we go with `undefined`)

Thanks 😄